README.md

```diff
@@ -3,6 +3,17 @@
 
 This is the C++ version of [frc-characterization](https://github.com/wpilibsuite/frc-characterization). It uses the [wpimath](https://github.com/wpilibsuite/allwpilib/tree/main/wpimath) backend for generating feedforward and feedback gains.
 
+## Supported Vendor Hardware
+This is a list of the vendors that are currently supported by SysId for 2022:
+
+### Vendors:
+- REV
+
+### Hardware Supported by WPILib:
+- Analog Gyro
+- PWM Motor Controllers
+- Encoders plugged into the roboRIO
```

Comment:
The other sections have a blank line after the heading. Also, the colon suffix is unnecessary.